src/app/pages/system/alert-config/alert-config.component.html

```diff
@@ -0,0 +1,48 @@
+<mat-card class="form-card">
+  <mat-progress-bar *ngIf="isFormLoading" mode="indeterminate"></mat-progress-bar>
+  <div class="mat-title">
+    <div *ngIf="selectedCategory"> {{ 'Category' | translate }}: {{ selectedCategory.title }} </div>
```

Comment:
Let's use a normal `ix-checkbox` with a label of `Category`. This doesn't look right. We can use a normal checkbox to present the currently selected category. I think that would look best.

Reply:
I meant `ix-select` here. My bad.